DirectProgramming/C++SYCL_FPGA/README.md

```diff
@@ -94,14 +94,13 @@ flowchart LR
-| [mmhost](Tutorials/Features/hls_flow_interfaces/mmhost)                                                                   | [Tutorials/Features/hls_flow_interfaces](Tutorials/Features/hls_flow_interfaces) | Basics of declaring Avalon memory-mapped host data interfaces for IP components
+| [mmhost](Tutorials/Features/hls_flow_interfaces/mmhost)                                                                       | [Tutorials/Features/hls_flow_interfaces](Tutorials/Features/hls_flow_interfaces) | Basics of declaring Avalon memory-mapped host data interfaces for IP components
 | [pipes](Tutorials/Features/pipes)                                                                                             | [Tutorials/Features](Tutorials/Features)                                                 | The basics of using SYCL*-compliant pipes extension for FPGA <br> How to declare and use pipes
 | [printf](Tutorials/Features/printf)                                                                                           | [Tutorials/DesignPatterns](Tutorials/DesignPatterns)                                     | How to declare and use `printf` in program
-| [invocation_interfaces](Tutorials/Features/hls_flow_interfaces/invocation_interfaces)                                     | [Tutorials/Features/hls_flow_interfaces](Tutorials/Features/hls_flow_interfaces) | How to specify the kernel invocation interface and kernel argument interfaces
-| [streaming_data_interfaces](Tutorials/Features/hls_flow_interfaces/streaming_data_interfaces)                             | [Tutorials/Features/hls_flow_interfaces](Tutorials/Features/hls_flow_interfaces) | How to use pipes to implement streaming data interfaces on an IP component
-| [parallel_loops](Tutorials/Features/task_sequence/parallel_loops)                                          | [Tutorials/Features](Tutorials/Features) | How to use task sequences to describe multiple parallel loops in a single kernel
-
-
+| [invocation_interfaces](Tutorials/Features/hls_flow_interfaces/invocation_interfaces)                                         | [Tutorials/Features/hls_flow_interfaces](Tutorials/Features/hls_flow_interfaces) | How to specify the kernel invocation interface and kernel argument interfaces
+| [streaming_data_interfaces](Tutorials/Features/hls_flow_interfaces/streaming_data_interfaces)                                 | [Tutorials/Features/hls_flow_interfaces](Tutorials/Features/hls_flow_interfaces) | How to use pipes to implement streaming data interfaces on an IP component
+| [parallel_loops](Tutorials/Features/task_sequence/parallel_loops)                                                             | [Tutorials/Features](Tutorials/Features) | How to use task sequences to describe multiple parallel loops in a single kernel

```

Comment:
Actually, these are supposedly order alphabetically, I see that some others are not correctly placed

Reply:
i did not put it in alphabetical order, I tends to add new tutorial to the end of the list. let me correct that